src/server/analytic_services/prometheus_reports/noobaa_endpoint_report.js

```diff
@@ -74,6 +75,42 @@ const NOBBAA_ENDPOINT_METRICS = js_utils.deep_freeze([
             labelNames: ['bucket_name']
         }
     },
+    {
+        type: 'Histogram',
+        name: 'hub_read_latency',
+        configuration: {
+            help: 'hub read latency in namespace cache bucket',
+            labelNames: ['namespace_resource_id'],
+            buckets: [0.1, 10, 100, 200, 500, 1000, 2000],
```

Comment:
Lets set the buckets of all latency histograms to something like this - added higher/lower values because this includes also small/large objects. 
```suggestion
            buckets: [
                0.001, 
                0.01, 
                0.1, 0.2, 0.5,
                1, 2, 5, 
                10, 20, 50, 
                100, 200, 500, 
                1000, 2000, 5000,
                10000, 20000, 50000,
            ],
```